src/exo/API_scheduling.py

```diff
@@ -1884,6 +1884,34 @@ def merge_writes(proc, block_cursor):
     return Procedure(ir, _provenance_eq_Procedure=proc, _forward=fwd)
 
 
+@sched_op([AssignOrReduceCursorA])
+def split_write(proc, stmt):
+    """
+    Split a reduce or assign statement with an addition on the RHS into two
+    writes.
+
+    This operation is the opposite of the last two cases of `merge_writes`.
+
+    args:
+        stmt    - cursor pointing to the assign/reduce statement.
+
+    rewrite:
+        `a = b + c`
+            ->
+        `a = b`
+        `a += c`
+        ----------------------
+        `a += b + c`
+            ->
+        `a += b`
+        `a += c`
+        ----------------------
+
```

Comment:
When adding scheduling ops, I think we should try to specify forwarding rules in documentation, or at the very least, write a few test cases in `tests/test_cursors.py` so that we understand the behavior. I think for this scheduling operation, 
- cursor to the statement (and any subtree cursors) gets invalidated.
- blocks containing the statement will forward naturally.

Reply:
Yeah that's a good idea. I will add both the tests and documentation.

Reply:
Done!